u8timeseries/timeseries.py

```diff
@@ -398,15 +401,17 @@ def from_times_and_values(times: pd.DatetimeIndex,
 
         return TimeSeries(series, series_lo, series_hi)
 
-    def plot(self, *args, plot_ci=True, **kwargs):
+    def plot(self, *args, plot_ci=True, new_plot=True, **kwargs):
         """
         Currently this is just a wrapper around pd.Series.plot()
         """
         # temporary work-around for the pandas.plot issue
         # errors = self._combine_or_none(self._confidence_lo, self._confidence_hi,
         #                                lambda x, y: np.vstack([x.values, y.values]))
         # self._series.plot(yerr=errors, *args, **kwargs)
-        fig, ax = plt.subplots()
+        global fig, ax
```

Comment:
Not a fan of keeping those as global, can backfire in other places. Would new_plot be enough?

Reply:
Fair enough, another possibility would be to return ax, and provide it as an optional argument to the next invocation of plot()

Reply:
As far as I can see, we need a way to hold on to the ax object to be able to plot on the same axis.

Reply:
You probably don't need to keep these globals yourself. You should be able to get the current Figure or Axes currently active in pyplot. I would keep the `new_plot` flag, default it to `False`, and then use such tricks (havent' tested):

```
if new_plot:
   fig, ax = plt.subplots()
else:
   fig, ax = plt.gcf(), plt.gca()
self._series.plot()
fig.autofmt_xdate()
```

Reply:
This fixes it, thanks!